include/oneapi/dpl/internal/scan_by_segment_impl.h

```diff
@@ -94,7 +94,7 @@ class __seg_scan_wg_kernel;
 template <bool __is_inclusive, typename... Name>
 class __seg_scan_prefix_kernel;
 
-template <bool __is_inclusive>
```

Comment:
Why `__BackendTag` is not a part of `operator()`? Looks inconsistent with everything else.

Reply:
Fixed